src/windows/views/effects_listview.py

```diff
@@ -59,7 +59,7 @@ def startDrag(self, event):
 
         if not current.isValid():
             # We can't find anything to drag
-            log.warning("No draggable items found in model!")
+            log.warning("No draggable items found in EffectsListView model!")
```

Comment:
Eh. This'll display with the module name prepended to it, so the log message will look like:
```
effects_listview:WARNING: No draggable items found in model!
```
Naming the class is kind of redundant. And it's the kind of thing that screws us up later, when someone cut-and-pastes that code to another module without updating the class name in the log message. (Yes, we end up coding defensively against the _stupidest_ potential problems sometimes...)

Reply:
Ah, I see. I'm sorry for causing more work. I'll undo it and all of the others :-)

Reply:
Not at all! Like I said, it's a _dumb_ problem to have to structure the code around. Hardly the sort of thing you'd expect to have to consider.

But, for better or for worse, copy-paste code duplication is also undeniably a major characteristic of our existing codebase. (I have, in my more cynical periods, [been known](https://github.com/OpenShot/openshot-qt/pull/2512#issuecomment-453385547) to hold it up as the _defining feature_ of our codebase.) :laughing: